### PR TITLE
Add support for native separators in oven, and vs2015/2017 FBX sdk

### PR DESCRIPTION
**Pull request by huffman**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1062/HighFidelity-Beta-PR10801-b12018ccf3f7cb2dd5e76cbf05ccb7a1e6d91fbc.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1062/HighFidelity-Beta-PR10801-b12018ccf3f7cb2dd5e76cbf05ccb7a1e6d91fbc.exe

File: cmake/modules/FindFBX.cmake [hunk 1]
```diff
@@ -56,6 +56,14 @@ elseif (${CMAKE_CXX_COMPILER_ID} MATCHES "GNU")
 endif()
 
 function(_fbx_find_library _name _lib _suffix)
+  if (MSVC15)
```
Comment (jherico) [inline cmake/modules/FindFBX.cmake]: not critical, but CMake discourages the use of these now, favoring [MSVC_VERSION](https://cmake.org/cmake/help/v3.8/variable/MSVC_VERSION.html#variable:MSVC_VERSION)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1098/HighFidelity-Beta-PR10801-5f92c1adc09abb7c332a300c40fc1b352c1c6257.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1098/HighFidelity-Beta-PR10801-5f92c1adc09abb7c332a300c40fc1b352c1c6257.dmg

**Comment by huffman** [comment]
This is an internal tool that I tested myself. Marking QA Approved.

**Pull request merged**